doc/classes/ProjectSettings.xml

```diff
@@ -102,7 +102,7 @@
 			<param index="0" name="name" type="StringName" />
 			<description>
 				Similar to [method get_setting], but applies feature tag overrides if any exists and is valid.
-				[b]Example:[/b]	If the setting override [code]"application/config/name.windows"[/code] exists, and the following code is executed on a [i]Windows[/i] operating system, the overridden setting is printed instead:
+				[b]Example:[/b] If the setting override [code]"application/config/name.windows"[/code] exists, and the following code is executed on a [i]Windows[/i] operating system, the overridden setting is printed instead:
```

Comment:
🤷

Reply:
This seems to replace `U+0009 Character Tabulation` with `U+0020 SPACE`, which is correct. No idea how that tab got there.

Reply:
I was the one that modified this line, but that tab shouldn't be allowed in the first place. [@Repiteo mentioned](https://github.com/godotengine/godot/pull/99031#pullrequestreview-2425888078) he wanted to take a look at preventing this in the future.
Because I didn't check for tabs there may be more like this.
And indeed there are tabs here.
![image](https://github.com/user-attachments/assets/ff2b6cbc-4d2e-4483-9b47-85e5c6209e93)